examples/specialty_plots/ishikawa.py

```diff
@@ -0,0 +1,229 @@
+"""
+=================
+Ishikawa Diagrams
+=================
+
+Ishikawa Diagrams, fishbone diagrams, herringbone diagrams, or cause-and-effect
+diagrams are useful for visualizing the effect to many cause relationships
+Source: https://en.wikipedia.org/wiki/Ishikawa_diagram
+
+"""
+import logging
+from typing import List
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import matplotlib
+
+_log = logging.getLogger(__name__)
+
+
+def drawspines(data: dict, ax: plt.Axes, parentspine: None, alpha_ps: float = 60.0,
+               alpha_ss: float = 0,
+               primary_spines_rel_xpos: List[float] = [np.nan],
+               spine_color: str = "black", reclevel: int = 1,
+               max_recursion_level: int = 4, debug: bool = False):
+    """
+    Draw an Ishikawa spine with recursion
+
+    Parameters
+    ----------
+    data : dict
+        dictionary structure containing problem, causes...
+    ax : plt.Axes
+        Matplotlib current axes
+    parentspine : None
+        Parent spine object
+    alpha_ps : float, optional
+        First spine angle using during recursion. The default is 60.0.
+    alpha_ss : float, optional
+        Secondary spine angle using during recursion. The default is 0.
```

Comment:
`alpha` is not a clear name, especially since that's the name for the colour transparency. These should be named something like:
```suggestion
    primary_spine_angle : float, optional
        First spine angle using during recursion. The default is 60.0.
    secondary_spine_angle : float, optional
        Secondary spine angle using during recursion. The default is 0.
```